languages/common-lisp/concepts/lists/about.md

```diff
@@ -1 +1,9 @@
-TODO: add information on lists concept
+[Lists][hyper-cons-as-list] are a very common data type in Common Lisp. They are made up of a sequence of [cons][../cons/about.md] cells. Each `car` is an element of the list and every `cdr` is a either the next cons cell or a terminating atom.
+
+A list which terminates with the empty list is called a "proper list".
+
+A list which terminates with an atom that is not th empty list is called a "dotted list" (based upon how it is printed: `(cons 'a 'b) ;=> (a . b)`).
+
+A list can also be circular if some cons cell in the list `cdr` of a later cons cell.
```

Comment:
Maybe you could show the somewhat funky syntax for that here? I don't know if that matters all too much though (this thing: `#1=(1 2 3 . #1#)`)

Reply:
I can mention it - but then we'll need to link the, as of yet, unwritten reader-macro concept.

Reply:
I like what you settled on here! It gives some more specific info without confusing them with the circular syntax for now :)

Reply:
To be pedantic - the syntax is not 'circular syntax' it is a general syntax for referencing values via reader macros. For example something like:

```lisp
(if #1=(assoc 'a alist) (incf #1))
```

(in this case just use a damn `let` - sheesh!)